src/Logger.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+*   Logger.cpp
+*   ----------------------
+*   Created on: 2023/10/09
+*   Author: Lankow
+*/
+#include <Arduino.h>
+#include "Logger.hpp"
+
+Logger::Logger(SDCardHandler* p_sdCardHandler) : m_sdCardHandler(p_sdCardHandler) {}
+
+String Logger::formatSysTime() {
+    long milliseconds = millis();
```

Comment:
Tutaj znowu można by było użyć bibliotek c++ do czasu które te konwersje mają wbudowane. Jeśli dobrze pamietam to std::chrono